File: ios/RNMBX/RNMBXCircleLayerViewManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_REMAP_MODULE(RNMBXCircleLayer, RNMBXCircleLayerViewManager, RCTViewManager)

// circle layer props
RCT_EXPORT_VIEW_PROPERTY(sourceLayerID, NSString)

// standard layer props
#include "CommonLayerProperties.H"

@end

File: ios/RNMBX/RNMBXCircleLayerViewManager.swift
@objc(RNMBXCircleLayerViewManager)
class RNMBXCircleLayerViewManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
  
    override func view() -> UIView! {
      let layer = RNMBXCircleLayer()
      layer.bridge = self.bridge
      return layer
    }
}
